renderer/index.js

```diff
@@ -1019,6 +1025,12 @@ function openTorrentContextMenu (infoHash) {
   menu.popup(remote.getCurrentWindow())
 }
 
+function openDownloadFolder (torrentSummary) {
+  var folder = path.parse(torrentSummary.name).name
+  var fullPath = path.join(torrentSummary.path, folder)
```

Comment:
There's actually already a function called `showFolder` that implements this logic. Let's just use that.